.github/workflows/builds.yml

```diff
@@ -48,7 +48,7 @@ jobs:
                  -G "${{ matrix.PLATFORM.GENERATOR }}"
     # Perform the build.
     - name: Build
-      run: cmake --build build --config ${{ matrix.BUILD_TYPE }}
+      run: cmake --build build --config ${{ matrix.BUILD_TYPE }} -- -j 2
```

Comment:
Perhaps it would be useful to set this to `$(nproc)`?

Reply:
You can use `$(sysctl -n hw.logicalcpu)` on MacOS.

Reply:
Can I also use that on linux (because this job runs on multiple different OS)? Or maybe I should just split the build command...

Reply:
> Can I also use that on linux (because this job runs on multiple different OS)? Or maybe I should just split the build command...

No, you can't! Fun!